app/binary-intro/page.tsx

```diff
@@ -0,0 +1,67 @@
+'use client';
+
+import React, { useState } from 'react';
+import { Typography, Stack, Button, Box } from '@mui/material';
+import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown';
+import ArrowDropUpIcon from '@mui/icons-material/ArrowDropUp';
+import AutorenewIcon from '@mui/icons-material/Autorenew';
+
+const Page = () => {
+  const [decimalNumber, setDecimalNumber] = useState(0);
+
+  const increment = () => {
+    setDecimalNumber(decimalNumber + 1);
+  };
+
+  const decrement = () => {
+    if (decimalNumber > 0) {
+      setDecimalNumber(decimalNumber - 1);
+    }
+  };
+
+  const reset = () => {
+    setDecimalNumber(0);
+  };
+
+  const binaryNumber = decimalNumber.toString(2);
+
+  return (
+    <Box
+      display="flex"
+      justifyContent="center"
+      width="100%"
+    >
+      <Stack direction="column" alignItems="center">
+        <Box>
+          <Stack direction="row" alignItems="center">
+            <Stack direction="column" spacing={1}>
+              <Button variant="contained" size="small" onClick={increment}>
+                <ArrowDropUpIcon />
+              </Button>
+              <Button variant="contained" size="small" onClick={decrement}>
+                <ArrowDropDownIcon />
+              </Button>
+            </Stack>
+            <Stack direction="column" alignItems="center">
+              <Typography variant="h6" style={{ margin: '0 16px' }}>
```

Comment:
We can actually have a style directly applied to a `Typography` component with `marginY="16px"`

Reply:
done